### PR TITLE
driverStateV2

### PR DESCRIPTION
**Pull request by adeebshihadeh**

### CODE REVIEW

File: log.capnp [hunk 1]
```diff
@@ -1614,7 +1614,36 @@ struct Joystick {
...
+    facePositionStd @3 :List(Float32);
+    faceProb @4 :Float32;
+    leftEyeProb @5 :Float32;
+    rightEyeProb @6 :Float32;
+    leftBlinkProb @7 :Float32;
+    rightBlinkProb @8 :Float32;
+    sunglassesProb @9 :Float32;
+    occludedProb @10 :Float32;
+    readyProb @11 :List(Float32);
+    notReadyProb @12 :List(Float32);
```
Comment (adeebshihadeh) [inline log.capnp:1642]: notReadyProb isn't 1 - readyProb?

File: log.capnp [hunk 2]
```diff
@@ -1614,7 +1614,36 @@ struct Joystick {
...
+  dspExecutionTime @2 :Float32;
+  rawPredictions @3 :Data;
+
+  poorVision @4 :Float32;
+  wheelOnRight @5 :Float32;
+
+  driverDataLH @6 :DriverData;
+  driverDataRH @7 :DriverData;
+
+  struct DriverData {
```
Comment (adeebshihadeh) [inline log.capnp:1629]: should we come up with a better name than driver?

File: log.capnp [hunk 3]
```diff
@@ -1614,7 +1614,36 @@ struct Joystick {
...
 
-struct DriverState {
+struct DriverStateV2 {
+  frameId @0 :UInt32;
+  modelExecutionTime @1 :Float32;
+  dspExecutionTime @2 :Float32;
+  rawPredictions @3 :Data;
+
+  poorVision @4 :Float32;
+  wheelOnRight @5 :Float32;
```
Comment (adeebshihadeh) [inline log.capnp:1624]: I assume this is a probability? add the Prob suffix for consistency?

File: log.capnp (hunk 1 above)
Comment (ZwX1616) [reply log.capnp:1642]: no they don't mean the same thing

File: log.capnp (hunk 2 above)
Comment (ZwX1616) [reply log.capnp:1629]: can use "occupant" instead? if that doesn't sound weird

File: log.capnp (hunk 3 above)
Comment (ZwX1616) [reply log.capnp:1624]: yeah

**Pull request merged**